<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Boanc Ionut-Silviu</author>
 <class>info_window</class>
 <widget class="QWidget" name="info_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CIIT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border-radius: 16px;
	padding-left: 8px;
	padding-right: 8px;
	padding-top: 8 px;
	padding-bottom: 8 px;
	color: rgb(190, 190, 190);
	background-color: rgb(55, 79, 91);
    }

QWidget {
 background-color: rgb(34, 48, 56);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>8</number>
   </property>
   <item row="1" column="2">
    <widget class="QPushButton" name="pb_contacts">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 16px;
	padding-left: 8px;
	padding-right: 8px;
	padding-top: 8 px;
	padding-bottom: 8 px;
	color:rgb(226, 220, 220);
	background-color: rgb(56, 76, 83);
    }</string>
     </property>
     <property name="text">
      <string>Contacts</string>
     </property>
     <property name="icon">
      <iconset resource="../icons/icons.qrc">
       <normaloff>:/Icons/user.svg</normaloff>:/Icons/user.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QLabel" name="lb_title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Vitesco</family>
       <pointsize>32</pointsize>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
    color: rgb(226, 220, 220);
}</string>
     </property>
     <property name="text">
      <string>Info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QPushButton" name="pb_back">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 16px;
	padding-left: 8px;
	padding-right: 8px;
	padding-top: 8 px;
	padding-bottom: 8 px;
	color:rgb(226, 220, 220);
	background-color: rgb(56, 76, 83);
    }</string>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
     <property name="icon">
      <iconset resource="../icons/icons.qrc">
       <normaloff>:/Icons/arrow-left.svg</normaloff>:/Icons/arrow-left.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_previous_page">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>24</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(34, 48, 56);
}
QPushButton:hover {
	background-color: rgb(30, 43, 50);
}
QPushButton:pressed {
background-color: rgb(24, 35, 40)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/Icons/arrow-left-circle.svg</normaloff>:/Icons/arrow-left-circle.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="sw_pages">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="styleSheet">
         <string notr="true">image: url(:/Info/info1.png);
</string>
        </property>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="styleSheet">
         <string notr="true">image: url(:/Images/overview.png);</string>
        </property>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="styleSheet">
         <string notr="true">image: url(:/Images/advanced.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_next_page">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>24</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(34, 48, 56);
}
QPushButton:hover {
	background-color: rgb(30, 43, 50);
}
QPushButton:pressed {
background-color: rgb(24, 35, 40)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/Icons/arrow-right-circle.svg</normaloff>:/Icons/arrow-right-circle.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
